PythonAPI/docs/client.yml

```diff
@@ -119,6 +120,31 @@
         Reload the current world, note that a new world is created with default
         settings using the same map. All actors present in the world will be
         destroyed, __but__ traffic manager instances will stay alive.
+      # --------------------------------------
+    - def_name: load_world_if_different
+      params:
+      - param_name: map_name
+        type: str
+        doc: >
+          Name of the map to be used in this world. Accepts both full paths and map names, e.g.
+          '/Game/Carla/Maps/Town01' or 'Town01'. Remember that these paths are dynamic.
+      - param_name: reset_settings
+        type: bool
+        default: true
+        doc: >
+          Option to reset the episode setting to default values, set to false to keep the current settings.
+          This is useful to keep sync mode when changing map and to keep deterministic scenarios.
+      - param_name: map_layers
+        type: carla.MapLayer
+        default: carla.MapLayer.All
+        doc: >
+          Layers of the map that will be loaded. By default all layers are loaded.
+          This parameter works like a flag mask.
+      return: carla.World
```

Comment:
Currently does not return a world object.
https://github.com/carla-simulator/carla/blob/ed4e0176f727c5fa5c5299ee7e8eb37064c5ed48/LibCarla/source/carla/client/Client.h#L93-L96